src/views/name.njk

```diff
@@ -5,6 +5,9 @@
 {% if props.parameters['which-type'] == "registerLp" %}
    {% include "pages/name/limited-partnership-name.njk" %}
 {% elif props.parameters['which-type'] == "registerPflp" %}
+  {% include "pages/name/private-limited-partnership-name.njk" %} 
+{% elif props.parameters['which-type'] == "registerSlp" %}
+  {% include "pages/name/scottish-limited-partnership-name.njk" %}
   {% include "pages/name/private-limited-partnership-name.njk" %}  
```

Comment:
This is wrong - maybe due to the merge?

There shouldn't be a second include in this `elif` block.

I'm getting this error in my logs, are you? (presumably related)

`limited-partnerships-web  | {"created":"2024-12-10T10:40:01.825+00:00","event":"error","namespace":"web-security-node","data":{"message":"uncaughtException: ⨯ Unable to compile TypeScript:\nsrc/infrastructure/gateway/RegistrationGateway.ts(65,44): error TS2551: Property 'patchLimitedPartnership' does not exist on type 'LimitedPartnershipsService'. Did you mean 'postLimitedPartnership'?\n\nTSError: ⨯ Unable to compile TypeScript:\nsrc/infrastructure/gateway/RegistrationGateway.ts(65,44): error TS2551: Property 'patchLimitedPartnership' does not exist on type 'LimitedPartnershipsService'. Did you mean 'postLimitedPartnership'?\n\n    at createTSError (/app/node_modules/ts-node/src/index.ts:859:12)\n    at reportTSError (/app/node_modules/ts-node/src/index.ts:863:19)\n    at getOutput (/app/node_modules/ts-node/src/index.ts:1077:36)\n    at Object.compile (/app/node_modules/ts-node/src/index.ts:1433:41)\n    at Module.m._compile (/app/node_modules/ts-node/src/index.ts:1617:30)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)\n    at Object.require.extensions.<computed> [as .ts] (/app/node_modules/ts-node/src/index.ts:1621:12)\n    at Module.load (node:internal/modules/cjs/loader:1208:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1024:12)\n    at Module.require (node:internal/modules/cjs/loader:1233:19)"}}`

Reply:
run `npm install` to solve your issue
sdk has been updated

Reply:
👍 second include is still an issue.

Reply:
second include removed